net.wz2100.wz2100.yaml

```diff
@@ -0,0 +1,160 @@
+app-id: net.wz2100.wz2100
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: warzone2100
+separate-locales: false
+finish-args: 
+  - --socket=wayland
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
```

Comment:
Does it need home access?

Reply:
For save files, configs, adding mods and other game data

Reply:
A, ok, just y view is only need indicated the home folder is used, in this case --filesystem=~/.warzone2100-3.2

Reply:
If anything, it should be `--persist=.warzone2100-3.2`. Please ask upstream to start supporting XDG directories.